pkg/kubelet/rkt/rkt.go

```diff
@@ -190,11 +191,31 @@ func (r *runtime) runCommand(args ...string) ([]string, error) {
 	return strings.Split(strings.TrimSpace(string(output)), "\n"), nil
 }
 
-// makePodServiceFileName constructs the unit file name for a pod using its UID.
-func makePodServiceFileName(uid types.UID) string {
+// makePodServiceFileName constructs the unit file name for a pod using
+// its pod UID and rkt UUID. We use the rkt UUID here because it's not
+// clear how to update and reload the service file once the service remains
+// after exited.
```

Comment:
@yujuhong Good question. So the situation here is 1 kube pod can map to 2 rkt pods (imagine the first rkt pod crashed, and we are about to start the second one).
For some reason(I don't recall now, but seems to be related to `RemainAfterExit`) it's not updated. Let's just hold one and let me just try `ReloadOrRestartUnit`.